sass/partials/_next_button.scss

```diff
@@ -1,29 +1,22 @@
 $next-button-height-min-sm: 80px;
 $next-button-height: 80px;
-$next-button-icon-color: $secondary_brown;
-$next-button-border-color: $secondary_brown_dark;
+$next-button-border-color: $secondary_brown;
+$next-button-icon-color: $next-button-border-color;

```

Comment:
is there a reason to assign one variable to another? why not just use `$next-button-border-color` wherever `$next-button-icon-color is used`?